IDL/appendable.idl

```diff
@@ -97,3 +97,8 @@ struct AppendableInheritanceStruct : AppendableShortStruct
 struct AppendableInheritanceEmptyStruct : AppendableShortStruct
 {
 };
+
+struct AppendableExtensibilityInheritance : AppendableShortStruct
+{
+    long var_long;
+};
```

Comment:
Should not the extensibility be explicit to avoid confusion? I suggest adding the extensibility annotation to every structure in the test IDLs. It might be done in another PR too.

Reply:
The case you are talking about is already tested. This case tests the inheritance when a derived structure 
does not have the extensibility defined but the base structure does.